### PR TITLE
Update contact point  to planet community SH forum 

### PR DESCRIPTION
**Pull request by Dorothyrono**
Changes: 
- Contact
- License

### CODE REVIEW

File: collections/sentinel-2-l1c.yaml [hunk 1]
```diff
@@ -27,7 +27,8 @@ Tags:
   - open data
   - sentinel
 License: |
-  [License](https://docs.sentinel-hub.com/api/latest/data/sentinel-2-l1c/#attribution-and-use)
+  - <a href="https://www.sentinel-hub.com/tou/" target="_blank">Sentinel Hub Terms of Use</a>.
+  - <a href="https://docs.sentinel-hub.com/api/latest/data/sentinel-2-l1c/#attribution-and-use" target="_blank">Attribution and Use </a>.
```
Comment (maximlamare) [inline collections/sentinel-2-l1c.yaml:31]: Attribution and Use > Sentinel-2 L1C Attribution and Use

File: collections/sentinel-3-l1b-olci.yaml [hunk 2]
```diff
@@ -31,7 +31,8 @@ Tags:
   - open data
   - sentinel
 License: |
-  [License](https://docs.sentinel-hub.com/api/latest/data/sentinel-3-olci-l1b/#attribution-and-use)
+  - <a href="https://www.sentinel-hub.com/tou/" target="_blank">Sentinel Hub Terms of Use</a>.
+  - <a href="https://docs.sentinel-hub.com/api/latest/data/sentinel-3-olci-l1b/#attribution-and-use" target="_blank">Attribution and Use </a>.
```
Comment (maximlamare) [inline collections/sentinel-3-l1b-olci.yaml:35]: Attribution and Use > Sentinel-3 OLCI Attribution and Use

File: collections/sentinel-3-l1b-slstr.yaml [hunk 3]
```diff
@@ -31,7 +31,8 @@ Tags:
   - open data
   - sentinel
 License: |
-  [License](https://docs.sentinel-hub.com/api/latest/data/sentinel-3-slstr-l1b/#attribution-and-use)
+  - <a href="https://www.sentinel-hub.com/tou/" target="_blank">Sentinel Hub Terms of Use</a>.
+  - <a href="https://docs.sentinel-hub.com/api/latest/data/sentinel-3-slstr-l1b/#attribution-and-use" target="_blank">Attribution and Use </a>.
```
Comment (maximlamare) [inline collections/sentinel-3-l1b-slstr.yaml:35]: Attribution and Use > Sentinel-3 SLSTR Attribution and Use

File: collections/sentinel-5p-l2-tropomi.yaml [hunk 4]
```diff
@@ -29,7 +29,8 @@ Tags:
   - open data
   - sentinel
 License: |
-  [License](https://docs.sentinel-hub.com/api/latest/data/sentinel-5p-l2/#attribution-and-use)
+  - <a href="https://www.sentinel-hub.com/tou/" target="_blank">Sentinel Hub Terms of Use</a>.
+  - <a href="https://docs.sentinel-hub.com/api/latest/data/sentinel-5p-l2/#attribution-and-use" target="_blank">Attribution and Use </a>.
```
Comment (maximlamare) [inline collections/sentinel-5p-l2-tropomi.yaml:33]: Attribution and Use > Sentinel-5P Attribution and Use

**Review by maximlamare — CHANGES_REQUESTED** [review]
A few points to review

File: collections/copernicus-dem.yaml [hunk 5]
```diff
@@ -29,7 +29,7 @@ Tags:
   - orthorectification
   - open data
 License: |
```
[the remark this answers is not in the document]
Comment (Dorothyrono) [reply collections/copernicus-dem.yaml:31]: It links to a page not found [here](https://spacedata.copernicus.eu/documents/20123/121286/CSCDA_ESA_Mission-specific+Annex_31_Oct_22.pdf/fb109818-56ad-bbee-053c-d972aed25ce6?t=1674741175657)   
probably best to link the [main page ](https://spacedata.copernicus.eu/collections/copernicus-digital-elevation-model) incase it gets updated

File: collections/cnes-land-cover-map.yaml [hunk 6]
```diff
@@ -26,7 +26,9 @@ BandInformation:
           Description: The confidence of the classifier.
         - Name: OCS_Validity
           Description: The number of cloudless images.
-Contact: "Jordi Inglada: jordi.inglada@cesbio.eu, Vincent Thierion: vincent.thierion@inrae.fr, Silvia Valero: silvia.valero@cesbio.cnes.fr"
+Contact: |
```
[the remark this answers is not in the document]
Comment (Dorothyrono) [reply collections/cnes-land-cover-map.yaml:29]: Yeap, not upto date   
best to link to the [main page](https://www.theia-land.fr/en/product/land-cover-map/) to get upto date contacts

File: collections/corine-land-cover-accounting-layers.yaml [hunk 7]
```diff
@@ -23,7 +23,9 @@ BandInformation:
       Rows:
         - Name: CLC_ACC
           Description: Corine Land Cover Accounting Layer 
-Contact: <a href="mailto:sdi@eea.europa.eu">European Environment Agency</a> 
+Contact: |
+  - <a href="https://community.planet.com/sentinel-hub-81" target="_blank">Sentinel Hub Forum on Planet Commmunity</a>
+  - sdi@eea.europa.eu
```
[the remark this answers is not in the document]
Comment (Dorothyrono) [reply collections/corine-land-cover-accounting-layers.yaml:28]: done

File: collections/sentinel-5p-l2-tropomi.yaml (hunk 4 above)
Comment (Dorothyrono) [reply collections/sentinel-5p-l2-tropomi.yaml:33]: done

File: collections/sentinel-3-l1b-slstr.yaml (hunk 3 above)
Comment (Dorothyrono) [reply collections/sentinel-3-l1b-slstr.yaml:35]: done

File: collections/sentinel-2-l2a.yaml [hunk 8]
```diff
@@ -27,7 +27,8 @@ Tags:
   - open data
   - sentinel
 License: |
-  [License](https://docs.sentinel-hub.com/api/latest/data/sentinel-2-l2a/#attribution-and-use)
+  - <a href="https://www.sentinel-hub.com/tou/" target="_blank">Sentinel Hub Terms of Use</a>.
+  - <a href="https://docs.sentinel-hub.com/api/latest/data/sentinel-2-l2a/#attribution-and-use" target="_blank">Attribution and Use </a>.
```
[the remark this answers is not in the document]
Comment (Dorothyrono) [reply collections/sentinel-2-l2a.yaml:31]: done

File: collections/sentinel-2-l1c.yaml (hunk 1 above)
Comment (Dorothyrono) [reply collections/sentinel-2-l1c.yaml:31]: done

**Review by maximlamare — APPROVED** [review]

**Pull request merged**